CycloneDX.Core/Services/ComponentService.cs

```diff
@@ -33,14 +33,14 @@ public ComponentService(INugetService nugetService)
         /// <summary>
         /// Recursively retrieve all components and their dependencies from NuGet packages
         /// </summary>
-        /// <param name="projectFilePath"></param>
+        /// <param name="nugetPackages"></param>
         /// <returns></returns>
-        public async Task<HashSet<Component>> RecursivelyGetComponentsAsync(IEnumerable<NugetPackage> nugetPackges)
+        public async Task<HashSet<Component>> RecursivelyGetComponentsAsync(IEnumerable<NugetPackage> nugetPackages)
         {
             var components = new HashSet<Component>();
 
             // Initialize the queue with the current packages
-            var packages = new Queue<NugetPackage>(nugetPackges);
+            var packages = new Queue<NugetPackage>(nugetPackages);
 
```

Comment:
This change has value. Please raise a separate PR for this @MCCshreyas.

Reply:
Separate PR?